### PR TITLE
update docs to reflect new SSC UI

### PR DESCRIPTION
**Pull request by khorne3**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team

### CODE REVIEW

File: docs/semgrep-supply-chain/overview.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ severity levels for GA languages. That means Semgrep Supply Chain can flag all
 your critical/high-severity findings as either reachable or unreachable.
   * If there's a code pattern in the codebase that matches the vulnerability
     definition, the finding is flagged as **reachable**.
+      * A finding is **always reachable** if it's something Semgrep recommends fixing, regardless of what's in the code. In other words, the presence of the dependency is the vulnerability.
```
Comment (s-santillan) [inline docs/semgrep-supply-chain/overview.md:63]:
```suggestion
      * A finding is **always reachable** if it's something Semgrep recommends fixing, regardless of what's in the code. In other words, the presence of the dependency is the vulnerability.
```
I checked the app and this is indeed what's in the tooltip. However, I'm pretty sure the original meaning of Always reachable is that there's no other way to fix the vuln except by upgrading (other reachable findings can be fixed by keeping the library but using a different function)

@andy-r2c Would you suggest we keep the upgrade-only bit in the docs? 
image

File: docs/semgrep-supply-chain/triage-remediation.md [hunk 2]
```diff
@@ -104,129 +91,49 @@ To assess your findings, Semgrep Supply Chain provides the following methods:
...
-    <li>No time to fix</li>
-    </ul>
-   </td>
-  </tr></tbody>
-</table>
+| Filter                 | Description  |
+| ---------------------  | ------------ |
+| **Projects**           | Filter by repositories connected to your Semgrep account. |
+| **Branch**             | Filter by findings in different Git branches. |
+| **Teams**              | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with RBAC enabled. |
```
Comment (s-santillan) [inline docs/semgrep-supply-chain/triage-remediation.md:100]:
```suggestion
| **Teams**              | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with [Teams](/deployment/teams#teams-beta) enabled. |
```
**Teams** is the term that is used in the app itself. AFAIK there's no mention of RBAC :)

File: docs/semgrep-supply-chain/triage-remediation.md [hunk 3]
```diff
@@ -40,33 +40,20 @@ Perform triage and remediation on your open source dependencies through the **Su
...
 
-![Semgrep Supply Chain Vulnerabilities page](/img/sc-vulnerabilities.png)
+![Semgrep Supply Chain Vulnerabilities page](/img/sc-vulns.png)
 _Figure 1_. Semgrep Supply Chain Vulnerabilities page.
 
-## Assessing and triaging dependency findings and usages
+## Assess and triage dependency findings and usages
 
 :::info Prerequisite
 At least one repository that scans for dependencies through Semgrep Supply Chain. See [Getting started with Semgrep Supply Chain](/semgrep-supply-chain/getting-started).
```
Comment (s-santillan) [inline docs/semgrep-supply-chain/triage-remediation.md:49]: Link text (title) needs updating to "scan third-party dependencies" (http://semgrep.dev/docs/semgrep-supply-chain/getting-started)

File: docs/semgrep-supply-chain/triage-remediation.md [hunk 4]
```diff
@@ -104,129 +91,49 @@ To assess your findings, Semgrep Supply Chain provides the following methods:
...
+| **Severity**           | Filter by the severity of a finding. Filters are based on the severity of a vulnerability. Semgrep Supply Chain rules use severity values set by the GitHub Advisory Database.  |
+| **Transitivity**       | Filter by the transitivity of the finding. |
+| **Reachability**       | Filter by exposure, or whether the finding is reachable or not. <ul><li>**Reachable**: A finding is reachable if there's a code pattern in the codebase that matches the vulnerability definition</li><li>**Always reachable**: A finding is always reachable if it's something Semgrep recommends fixing, regardless of what's in the code.</li><li>**Conditionally reachable**: A finding is conditionally reachable if Semgrep finds a way to reach it when scanning your code when certain conditions are met.</li><li>**Unreachable**: A finding is unreachable if you don't use the vulnerable piece of code of the library or package imported</li><li>**Undetermined**: A finding is undetermined if Semgrep Supply Chain determines that you use a vulnerable version of a dependency, but it doesn't have a relevant reachability rule</li></ul>|
```
Comment (s-santillan) [inline docs/semgrep-supply-chain/triage-remediation.md:105]:
```suggestion
| **Reachability**       | Filter by exposure, or whether the finding is reachable or not. <ul><li>**Reachable**: A finding is reachable if there's a code pattern in the codebase that matches the vulnerability definition</li><li>**Always reachable**: A finding is always reachable if it's something Semgrep recommends fixing, regardless of what's in the code.</li><li>**Conditionally reachable**: A finding is conditionally reachable if Semgrep finds a way to reach it when scanning your code when certain conditions are met.</li><li>**Unreachable**: A finding is unreachable if you don't use the vulnerable piece of code of the imported library or package.</li><li>**Undetermined**: A finding is undetermined if Semgrep Supply Chain determines that you use a vulnerable version of a dependency, but it doesn't have a relevant reachability rule.</li></ul>|
```

File: docs/semgrep-supply-chain/triage-remediation.md [hunk 5]
```diff
@@ -104,129 +91,49 @@ To assess your findings, Semgrep Supply Chain provides the following methods:
...
-   </td>
-  </tr></tbody>
-</table>
+| Filter                 | Description  |
+| ---------------------  | ------------ |
+| **Projects**           | Filter by repositories connected to your Semgrep account. |
+| **Branch**             | Filter by findings in different Git branches. |
+| **Teams**              | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with RBAC enabled. |
+| **Tags**               | Filter for findings based on the tags associated with the project. |
+| **Status**             | Filter the triage state of a finding: <ul><li>New: Vulnerabilities that have not undertaken triage or remediation action.</li><li>Fixed: Vulnerabilities that are no longer detected after a scan. This typically means that the dependency containing the vulnerability has been updated. Semgrep Supply Chain automatically checks if the dependency has been updated and sets the vulnerability's status as Fixed.</li><li>Ignored: Vulnerabilities that have been triaged as ignored by the user. </li></ul>|
```
Comment (s-santillan) [inline docs/semgrep-supply-chain/triage-remediation.md:102]:
```suggestion
| **Status**             | Filter the triage state of a finding: <ul><li>**New**: Vulnerabilities that have not undertaken triage or remediation action.</li><li>**Fixed**: Vulnerabilities that are no longer detected after a scan. This typically means that the dependency containing the vulnerability has been updated. Semgrep Supply Chain automatically checks if the dependency has been updated and sets the vulnerability's status as Fixed.</li><li>**Ignored**: Vulnerabilities that have been triaged as ignored by the user. </li></ul>|
```

File: docs/semgrep-supply-chain/overview.md (hunk 1 above)
Comment (andy-r2c) [reply docs/semgrep-supply-chain/overview.md:63]: I think we should change the language to be more "strongly suggest" than "you must"

**Review by s-santillan — APPROVED** [review]

**Pull request merged**